crsf.md

```diff
@@ -530,6 +532,24 @@ same as 0x16, but same conversion style as 0x17
 
 ## 0x27 Reserved
 
+## 0x2F RPM
+
+Frame type used to transmit RPM (revolutions per minute) telemetry data from the craft to the transmitter. This frame can be used to report motor or propeller RPM for monitoring performance or diagnostics.
+
+```cpp
+    uint8_t     rpm_source_id;  // Identifies the source of the RPM data (e.g., 0 = Motor 1, 1 = Motor 2, etc.)
```

Comment:
i understand the idea here, but this seems very generic.
is this something that would be agreeable to @pfeerick, @raphaelcoeffic and the rest of the @EdgeTX team?

Reply:
This is probably more a question for @raphaelcoeffic, @gagarinlg, @3djc to comment on than me. IMO though, generic is good, as it leaves it open for the most possible uses.

Reply:
Until we get a more sophisticated telemetry system, where we do not have to specify all sensor types in the protocol, I think this is a good way forward.
I support this.